src/rgw/support/rgw_support_bundle

```diff
@@ -166,8 +162,40 @@ class RGWSupportBundle:
         if coredumps:
             crash_dump_dir = os.path.join(log_dir, 'rgw_debug')
             if os.path.exists(crash_dump_dir):
-                shutil.copytree(crash_dump_dir,\
-                    os.path.join(RGWSupportBundle._tmp_src, 'rgw_debug'))
+                if size_limit:
+                    latest_5_cores = RGWSupportBundle._get_latest_files(
```

Comment:
again, dont put numbers (5 here) in var name, its bad approach. if number changes tomorrow, var name would be really misleading

Reply:
done